spec/features/adventures/user_views_adventure_spec.rb

```diff
@@ -87,8 +86,80 @@
     expect(page).not_to have_content(adventure.name)
   end
 
+  scenario "unauthenticated user successfully views list of public adventures " \
+  " by navigating to the all_public adventures page" do
+    # # bucket_list_1
+    # bucket_list_1 = FactoryGirl.create(:bucket_list, user_id: 1)
+    # adventure_1 = FactoryGirl.create(
+    #   :adventure,
+    #   user_id: 1,
+    #   is_shared: true
+    # )
+    # bucket_list_adventure_1 = FactoryGirl.create(
+    #   :bucket_list_adventure,
+    #   bucket_list_id: bucket_list_1.id,
+    #   adventure_id: adventure_1.id
+    # )
+    #
+    # # bucket_list_2
+    # bucket_list_2 = FactoryGirl.create(
+    #   :bucket_list,
+    #   user_id: 2,
+    #   title: "Mongolia",
+    #   is_shared: false
+    # )
+    # adventure_2 = FactoryGirl.create(
+    #   :adventure,
+    #   user_id: user.id,
+    #   name: "Sleep in a yurt"
+    # )
+    # bucket_list_adventure_2 = FactoryGirl.create(
+    #   :bucket_list_adventure,
+    #   bucket_list_id: bucket_list_2.id,
+    #   adventure_id: adventure_2.id
+    # )
+    #
+    # visit bucket_list_path(bucket_list_1.id)
+    #
+    # expect(page).to have_content(bucket_list_1.title)
+    # expect(page).to have_content(adventure_1.name)
+    # expect(page).not_to have_content(bucket_list_2.title)
+    # expect(page).not_to have_content(adventure_2.name)
+  end
+
   scenario "authenticated user successfully clicks on an adventure link and " \
   "navigates to the associated address" do
+    user = FactoryGirl.create(:user)
+    visit new_user_session_path
+    fill_in "Email", with: user.email
+    fill_in "Password", with: user.password
+    click_button "Log in"
+
+    bucket_list = FactoryGirl.create(:bucket_list, user_id: user.id)
+    adventure = FactoryGirl.create(
+      :adventure,
+      user_id: user.id
+    )
+    bucket_list_adventure = FactoryGirl.create(
```

Comment:
Useless assignment to variable - `bucket_list_adventure`. Did you mean `bucket_list`?